README.md

```diff
@@ -113,15 +155,53 @@ If you are interested in working on the implementation for an 'active'
 RFC, but cannot determine if someone else is already working on it,
 feel free to ask (e.g. by leaving a comment on the associated issue).
 
-## Reviewing RFC's
-
-Each week the [core team] will attempt to review some set of open RFC
-pull requests.
-
-We try to make sure that any RFC that we accept is accepted at the
-Friday team meeting, and reported in [core team notes]. Every
-accepted feature should have a core team champion, who will represent
-the feature and its progress.
+## For Core Team Members
+
+### Reviewing RFC's
+
+Each core team is responsible for reviewing open RFCs. If an RFC 
+requires work from their team, ensuring that the RFC reflects that. 
```

Comment:
```suggestion
requires work from their team, the RFC reflects that. 
```

Reply:
Removing those words changes the meaning here, but I'll update to make my intention more clear (it was a sloppy sentence to begin with 😬).

Something like:
"Each team must ensure that any RFC that requires work from their team appropriately states that fact"